e2e/clients/dashboard-transloadit/generateSignatureIfSecret.js

```diff
@@ -0,0 +1,34 @@
+const enc = new TextEncoder('utf-8')
+async function sign (secret, body) {
+  const algorithm = { name: 'HMAC', hash: 'SHA-384' }
+
+  const key = await crypto.subtle.importKey('raw', enc.encode(secret), algorithm, false, ['sign', 'verify'])
```

Comment:
[`Crypto.subtle`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle) is only available on `https` connections, we use `http` in tests.